src/app/(dashboard)/layout.tsx

```diff
@@ -9,14 +9,14 @@ export default async function DashboardLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en">
+    <div>
```

Comment:
Are you sure?

Reply:
We already have the html tag on the root layout, I believe this is what was causing the issues with dark mode and next-theme. Will need your help to test and verify 🙏